qa/tasks/mgr/dashboard/test_api.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+from .helper import DashboardTestCase
+
+DEFAULT_VERSION = '1.0'
+
+class VersionReqTest(DashboardTestCase):
+    def test_with_version(self): # This is the only one that should succeed
+        self._get('/api/summary', version=DEFAULT_VERSION)
+        self.assertStatus(200)
+        self.assertHeaders({
+            'Content-Type': 'application/vnd.ceph.api.v{}+json'.format(DEFAULT_VERSION),
+        })
+
+    def test_no_version(self): # Should fail
+        self._get('/api/summary', version="none")
+        self.assertStatus(415)
+
+    def test_with_wrong_version(self): # Should fail
+        self._get('/api/summary', version='99.99.1')
+        self.assertStatus(415)
```

Comment:
For test-cases that are parameterized, you can use [`unittest.subTest()`](https://docs.python.org/3.6/library/unittest.html#distinguishing-test-iterations-using-subtests):
```suggestion
    def test_version(self): # This is the only one that should succeed
        for (version, expected_status) in [
              (DEFAULT_VERSION, 200),
              (None, 415),
              ("99.99", 415)
            ]:
            with subTest(version=version):
                 self._get('/api/summary', version=version)
                 self.assertStatus(expected_status)
```

This way it won't exit the current test case if some subtest fails.